src/Driver/WpcliDriver.php

```diff
@@ -367,11 +367,16 @@ public function exportDatabase()
     /**
      * Import WordPress database.
      *
-     * @param string $import_file Relative path and filename of SQL file to import.
+     * If $import_file begins with a directory separator or ~ it is treated as an absolute path.
+     * Otherwise, it is treated as relative to the current working directory.
+     *
+     * @param string $import_file Relative or absolute path and filename of SQL file to import.
      */
     public function importDatabase($import_file)
     {
-        $import_file = getcwd() . "/{$import_file}";
+        if (! in_array($import_file[0], [DIRECTORY_SEPARATOR, '~'])) {
```

Comment:
There's a third parameter on `in_array` that controls whether it does a type-strict comparison (by default it doesn't). Let's set it true -- good habit to get into, so it doesn't trip us up somewhere in the future where it does matter -- and merge this!